src/commands/github/createOctokitClient.ts

```diff
@@ -14,6 +14,12 @@ export async function createOctokitClient(context: IActionContext & Partial<ISta
     return new Octokit(
         {
             userAgent: appendExtensionUserAgent(),
-            auth: token
+            auth: token,
+            /**
+             * 'baptiste' preview needed to work with template repos
+             * see https://docs.github.com/en/rest/overview/api-previews#create-and-use-repository-templates
+             * and https://developer.github.com/changes/2019-07-16-repository-templates-api/
+             */
+            previews: ['baptiste']
```

Comment:
I don't think it's a huge concern, but just wanted to bring it up to give it a proper consideration:
> API previews let you try out new APIs and changes to existing API methods before they become part of the official GitHub API.
>
> During the preview period, we may change some features based on developer feedback. If we do make changes, we'll announce them on the developer blog without advance notice.

I think that the preview tag `mercy-preview` (Repository topics) would be the only one I'd be wary of.  If we are worried about this, we can always have a separate Octokit specifically for creating from template.

Also their preview names are pretty funny.  Both of the ones I looked at were Overwatch specific 🤣

Reply:
Coming back to this... Just curious why you are concerned about using the mercy-preview?